library/components/TCsvExport.vue

```diff
@@ -36,13 +36,17 @@ export default {
   methods: {
     download() {
       this.is_loading = true;
-      payload = {
+      const payload = {
         layer: this.tLayer,
       };
-      let filters = this.getFiltersState ? this.getFiltersState : {};
+      let filters = this.getFiltersState() ? this.getFiltersState() : [];
+
+  const formattedFilters = filters.reduce((acc, filter) => {
+    return {...acc, [filter.name]: filter.value}
+  }, {});
```

Comment:
We are missing some spaces here, indentation won't be an issue when we move this project to a proper repo+project. Sorry about it

Reply:
I can't believe I missed that, thanks!